### ISSUE TITLE
Version control management

### ISSUE THREAD

**Issue by schmelz21** [cisagov/pe-reports#29]
# 💡 Summary #

P&E is currently looking to incorporate version control as per [cisagov dev-standards](https://github.com/cisagov/development-guide). Two report types are in current development, but 1 will eventually merge into `develop` branch and one to be archived as a "Release".

- [PE Report 0.1.1](https://github.com/cisagov/pe-reports/projects/1): This version is an active report delivery system for P&E. The current objective is to get the code up to dev-standards, add to the [cisagov/pe-reports](https://github.com/cisagov/pe-reports) repo and then archive as a pe-report "Release". To manage and access this version a  branch `pe-reports 0.1.1` will be created and used for any PE team member do commits/pull request pertaining to this version.

- [PE Report 1.0.0](https://github.com/cisagov/pe-reports/projects/2): This version is currently in development and introduces a whole new graphing system. The current objective is to continue development with the intention to merge into the main branch `develop` and act as the current P&E report moving forward and would be archived accordingly as updates occur. 

## Motivation and context ##

Priority is to be compliant to dev-standards and update the [cisagov/pe-reports](https://github.com/cisagov/pe-reports) to manage _**all code deemed important**_ to P&E reporting capabilities.

This would be useful because it allows for improved source control for the P&E team.

## Implementation notes ##

PE Report 0.1.1
- Create branch `pe-reports 0.1.1`
- P&E team members will create individual branches per issue/commits
- Pull Request will be set to "Draft Mode" as it is expected early PRs may not pass all checks.
- Upon 'pe-reports 0.1.1' is compliant and complete create a Release

PE Report 1.0.0
- P&E team members will create individual branches per issue/commits
- PR will follow [cisagov dev-standards](https://github.com/cisagov/development-guide)

## Acceptance criteria ##

How do we know when this work is done?

- [x] Create branch 'pe-reports 0.1.1'
- [ ]  'pe-reports 0.1.1' is set as Release

**Comment by schmelz21** [comment]
@jsf9k, @dav3r or @mcdonnnj - Please review. 
As P&E become more involved, we have a couple of moving parts to address in updating the pe-reports repo. I would like to confirm that our strategy is inline with expectations. If acceptable, let me know if there are any other procedural issues that need to be considered. Thanks.

**Comment by jsf9k** [comment]
* Personally I'd call the 0.1.1 branch `master` and the 1.0.0 branch `develop`, making `master` the default branch.  This nomenclature is more standard.
* If someone is working on a feature or bugfix for `master` (`develop`) then he or she should create a feature- or bugfix-specific branch for it from the latest `master` (`develop`).  Once the PR for the feature is merged into `master` (`develop`) the branch can be deleted.  _Long-lived branches other than `develop` and `master` (such as the developer-specific branches I think you are describing) are an antipattern._ 
* I'd suggest including more merge points.  One big merge at the end is going to be painful.
* Please read https://guides.github.com/introduction/flow/ for the basics of using feature branches.
* Please also read https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow which discusses the `develop` and `master` branch interactions in more detail.

@cisagov/team-ois, please chime in if you disagree with anything I've said, have things to add, etc.

**Comment by schmelz21** [comment]
@jsf9k makes sense and on same page. Originally, was hesitant to include the use of `master`. I will hold off a bit before we make adjustments in case someone else chimes in. We are still in position to work as this is resolved. As far as merges, they will be very small in scope moving forward. Needed to do a little catch-up. Thanks

**Comment by schmelz21** [comment]
Updated `pe-reports 0.1.1` to `master` branch and continuing to update repo in accordance to https://github.com/cisagov/pe-reports/issues/29 objectives. Acceptance criteria now invalid.
 
 Closing Issue.

**Comment by mcdonnnj** [comment]
@schmelz-ctr Sorry to raise something on an issue you just closed. If it's not too much trouble I think we should use `main` instead of `master` to mirror inclusive language changes seen industry wide.

**Comment by schmelz21** [comment]
@mcdonnnj not a problem. I can update to `main`.

**Comment by schmelz21** [comment]
Branch now is named `main`.
